.github/ISSUE_TEMPLATE/other-issue.md

```diff
@@ -0,0 +1,27 @@
+---
+name: Other issue
+about: Use this issue template to describe a current issue (that is not a new example analysis)
```

Comment:
```suggestion
about: Use this issue template to describe a current issue with an analysis or documentation (that is not a new example analysis)
```

We may want to add this here because `issue` alone may be a bit misleading in that one may think this template is to describe an existing issue/ticket rather than an issue with an existing analysis. What do you think?

Reply:
Note that I added "or documentation" here because I realized just "Use this issue template to describe a current issue with an analysis" may be too specific. However, this is not completely necessary? What do you think?

Reply:
I agree. Will use your wording.